<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolutionControl</class>
 <widget class="QDialog" name="SolutionControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solution Control and Output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Solver Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tolerance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum Iteration</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="solveCombo">
         <item>
          <property name="text">
           <string>Direct Solver</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="toleranceLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="maximumIterationLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="nodalSolutionBox">
     <property name="text">
      <string>Nodal Solution</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="elementSolutionBox">
     <property name="text">
      <string>Element Stress (Discontinue)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="averageElemenBox">
     <property name="text">
      <string>Average Element Stress</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="materialParameterBox">
     <property name="text">
      <string>Material Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
